src/einsteinpy/tests/test_metric/test_kerr.py

```diff
@@ -123,7 +123,7 @@ def test_calculate_trajectory3():
             [0 * u.m / u.s, 0.1 * u.rad / u.s, 951.0 * u.rad / u.s],
             0 * u.s,
             4e24 * u.kg,
-            0.33,
+            2e-3,
```

Comment:
Why did you change these?

Reply:
@ritzvik  made a comment about this above.
I shouldn't have changed these?

Reply:
It was requested by me. Earlier the values were not taking into consideration the max spin factor.

Reply:
Seems like @ritzvik Asked you to change it! :) Cool

Reply:
> @ritzvik made a comment about this above.
> I shouldn't have changed these?

Conveyed this to @shreyasbapat . No problem.